src/wp-includes/block-supports/layout.php

```diff
@@ -851,6 +911,25 @@ function wp_render_layout_support_flag( $block_content, $block ) {
 	return $processor->get_updated_html();
 }
 
+/*
+ * Add a `render_block_data` filter to fetch the parent block layout data.
+ */
+add_filter(
+	'render_block_data',
+	function ( $parsed_block, $source_block, $parent_block ) {
```

Comment:
Just double-checked the WP coding standards, and it looks like [it's discouraged to use anonymous functions in filters](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#closures-anonymous-functions):

> Closures should not be passed as filter or action callbacks, as removing these via remove_action() / remove_filter() is complex (at this time) (see [#46635](https://core.trac.wordpress.org/ticket/46635) for a proposal to address this).

Should we move this to a named function so that plugins (if they really wanted to) are able to `remove_filter` this?

Reply:
Done!